dbt/task/debug.py

```diff
@@ -1,58 +1,279 @@
+# coding=utf-8
+import os
+import platform
 import pprint
+import sys
 
 from dbt.logger import GLOBAL_LOGGER as logger
 import dbt.clients.system
 import dbt.config
 import dbt.utils
 import dbt.exceptions
+from dbt.adapters.factory import get_adapter
+from dbt.version import get_installed_version
+from dbt.config import Project, Profile
+from dbt.clients.yaml_helper import load_yaml_text
+from dbt.ui.printer import green, red
 
 from dbt.task.base_task import BaseTask
 
 PROFILE_DIR_MESSAGE = """To view your profiles.yml file, run:
 
 {open_cmd} {profiles_dir}"""
 
+ONLY_PROFILE_MESSAGE = '''
+A `dbt_project.yml` file was not found in this directory.
+Using the only profile `{}`.
+'''.lstrip()
+
+MULTIPLE_PROFILE_MESSAGE = '''
+A `dbt_project.yml` file was not found in this directory.
+dbt found the following profiles:
+{}
+
+To debug one of these profiles, run:
+dbt debug --profile [profile-name]
+'''.lstrip()
+
+COULD_NOT_CONNECT_MESSAGE = '''
+dbt was unable to connect to the specified database.
+The database returned the following error:
+
+  >{}
+
+Check your database credentials and try again. For more information, visit:
+https://docs.getdbt.com/docs/configure-your-profile
```

Comment:
i don't know how important this is, but we have a module for links: https://github.com/fishtown-analytics/dbt/blob/dev/grace-kelly/dbt/links.py

Do you think it's worth using here?